script/travis

```diff
@@ -24,6 +24,5 @@ MIX_ENV=test mix coveralls.travis
 
```

Comment:
line-13 of this file should be
source "$HOME/.bashrc"

Reply:
The required linting has been done using shellcheck.

Reply:
very much so. There are too many commits in this PR. Performing a squash and merge would be ideal here.